app/src/ui/lib/text-box.tsx

```diff
@@ -228,4 +260,16 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
       </div>
     )
   }
+
+  private onFocus = (event: React.FocusEvent<any>) => {
```

Comment:
Curious why we use `HTMLInputElement` over `FocusEvent`. Here's what the [docs](https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent) say on it.

Reply:
@iAmWillShepherd not sure I understand. My comment was a suggestion to avoid using `any` unless where we can be more explicit.

Reply:
oh, gotcha. 👍

Reply:
I actually talked with niik about my usage of any in the situations a couple of weeks back. I settled on any because we don't really care about the event. Went ahead and updated it tho.

Reply:
> I settled on any because we don't really care about the event.

If you really don't care about using the event, you could just drop the `event` parameter, e.g:

```
private onFocus = () => {
  if (!this.props.autoFocus && this.props.onFocus !== undefined) {
    this.props.onFocus()
  }
}
```